docs/install.md

```diff
@@ -16,16 +16,15 @@ You can run Compose on OS X and 64-bit Linux.  It is currently not supported on
 the Windows operating system. To install Compose, you'll need to install Docker
 first.
 
-Depending on how your system is configured, you may require `sudo` access to
-install Compose. If your system requires `sudo`, you will receive "Permission
-denied" errors when installing Compose. If this is the case for you, preface the
-install commands with `sudo` to install.
+> Note: If you get a "Permission denied" error, your `/usr/local/bin` directory
+probably isn't writable and you'll need to install Compose as the superuser. Run
+`sudo -i`, then the two commands above, then `exit`.
```

Comment:
@thaJeztah I guess we could break up the installation sections into "Docker Toolbox" (steps 1 & 2), then "Docker Compose" (formerly steps 3 onwards). Then add the original `sudo` text. See example: https://gist.github.com/charleswhchan/a0ef3c0a88bde8bd39bc

 ... let me know if you reach a decision. I can also back out the changes to install.md and only keep index.md for this PR. :grin: